beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md

```diff
@@ -6,7 +6,21 @@ The integration with Adobe Analytics provides a richer view of SDK events to use
 
 ![](../../../.gitbook/assets/aa-loop.gif)
 
-If the SDK made a network request with Adobe Analytics \(queued\), and if post-processing information is returned from Adobe Analytics about that event \(validated\), the status tells you when an SDK event is generated \(processed\).
+### Post-processed status
+
+If the SDK made a network request with Adobe Analytics, and if post-processing information is returned from Adobe Analytics about that event, the status tells you when an SDK event is generated.
+
+| Status | Description |
+| :--- | :--- | 
+|`Queued`|Network request is being made to fetch the post-processing information.|
+|`Processed`|Network request is successful and post-processing information is received.|
```

Comment:
"The network request was successful, and the post-processing information is received."